Warden is unable to deploy its commands or start, incorrect import directory

**Describe the bug**
What isn't working as intended, and what does it affect?
Affects core cunctionality
Bot is not able to start up, not able to deploy its commands.

**Affected versions**
What versions are affected by this bug? (e.g. >=3.0.1, 2.5.1-2.6.3, >=1.2.0)
2.0.0-next.1^

**Steps to reproduce**
Steps to reproduce the behavior. (e.g. click on a button, enter a value, etc. and see error)
1. Type in cli or terminal 
`node deploy-commands.js`

**Expected behavior**
What is expected to happen?
Show `Successfuly registered application commands`

**Actual behavior**
What actually happens? Attach or add errors or screenshots here as well.
avatar command is not using `shared.js`

```js
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module '../data.js'
Require stack:
- C:\Users\Ava\Documents\GitHub\Warden\commands\avatar.js
- C:\Users\Ava\Documents\GitHub\Warden\deploy-commands.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Module.require (node:internal/modules/cjs/loader:999:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (C:\Users\Ava\Documents\GitHub\Warden\commands\avatar.js:5:23)     
    at Module._compile (node:internal/modules/cjs/loader:1099:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1153:10)
    at Module.load (node:internal/modules/cjs/loader:975:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:999:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'C:\\Users\\Ava\\Documents\\GitHub\\Warden\\commands\\avatar.js',
    'C:\\Users\\Ava\\Documents\\GitHub\\Warden\\deploy-commands.js'
  ]
}
```

--- a/commands/avatar.js
+++ b/commands/avatar.js
@@ -2,7 +2,7 @@ const { SlashCommandBuilder } = require('@discordjs/builders');
 const { MessageEmbed } = require('discord.js');
 const { defaultColor, defaultLocale } = require('../settings.json');
 const { getLocale } = require('../functions.js');
-const { guildData } = require('../data.js');
+const { guildData } = require('../shared.js');
 
 module.exports = {
 	data: new SlashCommandBuilder()

